sql/catalyst/src/main/scala/org/apache/spark/sql/types/DateType.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.spark.sql.catalyst.ScalaReflectionLock
  *
  * Please use the singleton [[DataTypes.DateType]].
  *
- * Internally, this is represented as the number of days from epoch (1970-01-01 00:00:00 UTC).
+ * Internally, this is represented as the number of days from 1970-01-01.
```

Comment:
The DateType here has nothing with timezone, timezone is considered only when DateType is converted to/from TimestampType, right?